models/ed/DESCRIPTION

```diff
@@ -1,7 +1,7 @@
 Package: PEcAn.ED2
 Type: Package
 Title: PEcAn Package for Integration of ED2 Model
-Version: 1.7.2
+Version: 1.7.2.9000
```

Comment:
minor detail, but why `*.9000` instead of `*.1` 

or `1.7.3`, which I think is appropriate way to increment a bug fix following https://semver.org/?

Reply:
I'm used to bumping the version in pull requests with the development version (`usethis::use_version("dev")`) and only incrementing the patch or minor version for releases.  I think I noticed there are a couple other PEcAn packages with a version like 1.7.2.900x, so I figured maybe that was the way to go?

Reply:
From @infotroph on Slack:
>I like the tidyverse approach where releases are x.y.z, the dev version is always x.y.z.9000 with no increments between releases, and the next bump happens at release